sdk/src/main/java/io/radar/sdk/util/RadarLogBuffer.kt

```diff
@@ -0,0 +1,23 @@
+package io.radar.sdk.util
```

Comment:
any specific reason to introduce a `utils` package?  Currently all of the code is in a single package which keeps things really simple.

Reply:
Let's chat about this offline. I'd prefer to get away from the single-package approach to focus on organization as the SDK grows.

Reply:
I agree with @phil-brown, especially since it’s trivial and natural to break up packages in the Java/Kotlin world. In iOS, it’s more complicated, and often not worth bothering.